lib/Wrapper/OptimizingScheduler.cpp

```diff
@@ -564,6 +592,21 @@ bool ScheduleDAGOptSched::isOptSchedEnabled() const {
   }
 }
 
+bool ScheduleDAGOptSched::isTwoPassEnabled() const {
+  // check scheduler ini file to see if two pass scheduling is enabled
+  auto twoPassOption =
+      SchedulerOptions::getInstance().GetString("USE_TWO_PASS");
+  if (twoPassOption == "YES") {
+    return true;
+  } else if (twoPassOption == "NO") {
+    return false;
+  } else {
```

Comment:
llvm_unreachable

Reply:
Does it return an error or are you suggesting to call llvm_unreachable("<Error log>") instead of an else statement?

Reply:
I think we should have errors for unrecognized options. The message right now will only print with the debug flag passed to llvm. I think it's to easy to make mistakes with sched.ini options so we should tighten things up.

https://github.com/llvm/llvm-project/blob/905b002c139f039a32ab9bf1fad63d745d12423f/llvm/lib/Target/AMDGPU/SIInstrInfo.cpp#L4117

Reply:
I agree. I have changed it to output an error in the latest commit.